configs/confTypes/logtypes.go

```diff
@@ -0,0 +1,47 @@
+/*
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+package confTypes
+
+type pkg struct {
+	Name     string
+	LogLevel string
+}
+
+type accessLog struct {
+	LogFile string
+	Format string
+}
+
+// The log configuration struct.
+type LogConfig struct {
+
```

Comment:
Any reason for this empty line here? related to go specific formatting?

Reply:
This is already fixed